### DISCUSSION TITLE
Redirect after FormLoginAuthenticator not working

### DISCUSSION THREAD

**Discussion by JorgeMate** [symfony/symfony#54399] [Q&A] [answered]
Hello everybody. I am new here and this is my first request. I would thank anybody able to give me advice. 

My standard login form (FormLoginAuthenticator) works perfectly on local server, it redirects where I want after authentication, but not in the production server: after authentication, it always remain at the login form. I checked everything but no clue. 

I detected that the PHPSESID is changed at login in local server and then redirects, but in the remote server is does not change, nor redirect despite It authenticates the user.  

Im using a db to store the user sessions, trying to mitigate the fact that my NGINX remore server is behind a load balancer and using https. I ve been a few days already engaged with this. Any idea of what shall I do?

**Comment by MatTheCat** [comment]
Hello,

I guess there are many things that could go wrong, so [a reproducer](https://symfony.com/doc/current/contributing/code/reproducer.html) would be needed to investigate.

I will still attempt a shot in the dark though: wouldn’t the production user be denied access because of a missing role e.g., thus being redirected to the login page?

**Reply by JorgeMate** [reply, marked as answer]
I finally got it, MatTheCat, the same solution as in #54307: My prod server is behind a load balancer, so: 
**trusted_proxies: '127.0.0.1,REMOTE_ADDR'**
I´m so happy !!!
And thanks for your early shot !
